src/configurations/destinations/google_adwords_enhanced_conversions/db-config.json

```diff
@@ -4,8 +4,6 @@
   "config": {
     "auth": {
       "type": "OAuth",
-      "provider": "Google",
-      "role": "google_adwords_enhanced_conversions_v1",
       "rudderScopes": ["delivery"]
```

Comment:
Is `rudderScopes` in use for v1 route flow of rudder-auth too or dependency in data plane ?

Reply:
there is dependency in dataplane we need to clean that up